book/tutorials/jupyter.md

```diff
@@ -94,18 +94,18 @@ state of the repository including the complete history.
       This should be replaced with your GitHub username.
       <br/><br/>
       This step will also create a folder in the current location of your
-      terminal. The folder will be called `website`, which originates from the
+      terminal. The folder will be called `website2022`, which originates from the
```

Comment:
```suggestion
      terminal. The folder will be called `website`, which originates from the
```

Reply:
Re-generalize these?

Reply:
I think we need some sort of general naming pattern for each Hackweek. SnowEx used `website`, IceSat-2 used 'website2022`. Happy to leave the year in there and update this for each event. 
How about using `website-YYYY` for the template and make it a TODO for the clone guide?

Reply:
websiteYYYY i think would be fine to enforce. There was an issue doing variable substitution within backticks to allow for any repository name